src/Microsoft.AspNetCore.Mvc.Abstractions/Microsoft.AspNetCore.Mvc.Abstractions.csproj

```diff
@@ -1,6 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <Import Project="..\..\build\common.props" />
+  <Import Project="..\..\build\sourcelink.props" />
```

Comment:
Tested: `common.props(12,41): error MSB4099: A reference to an item list at position 3 is not allowed in this condition " '@(OtherProjectsToPack)' != '@(ProjectsToPack)' ".`